src/link/interfaces.ts

```diff
@@ -73,8 +73,9 @@ export interface LinkProps extends BaseComponentProps {
 
   /**
    * Called when a link is clicked without any modifier keys.
-   * Use this event if you want to prevent default browser navigation
-   * (by calling `preventDefault`) and implement client-side routing yourself.
+   *
+   * If you want to implement client-side routing yourself,
+   * use this event and prevent default browser navigation (by calling `preventDefault`).
```

Comment:
should we be more specific and say: "(by calling `event.preventDefault`)"

Reply:
I'm not sure if that will bring any more clarity, since the parameter will be named by the implementing team. It could be `event`, `e`, or something else.